apps/src/redux/instructions.js

```diff
@@ -90,50 +91,40 @@ export default function reducer(state = instructionsInitialState, action) {
   }
 
   if (action.type === TOGGLE_INSTRUCTIONS_COLLAPSED) {
-    return Object.assign({}, state, {
-      collapsed: !state.collapsed
-    });
+    return state.set('collapsed', !state.collapsed);
   }
 
   if (action.type === SET_INSTRUCTIONS_RENDERED_HEIGHT) {
-    return Object.assign({}, state, {
+    return state.merge({
```

Comment:
Integrating an Immutable Record into an existing redux module is surprisingly easy.